connectors/src/connectors/google_drive/index.ts

```diff
@@ -7,16 +7,14 @@ import {
   registerWebhook,
 } from "@connectors/connectors/google_drive/lib";
 import { ConnectorPermissionRetriever } from "@connectors/connectors/interface";
+import { Connector, ModelId, sequelize_conn } from "@connectors/lib/models";
 import {
-  Connector,
   GoogleDriveConfig,
   GoogleDriveFiles,
   GoogleDriveFolders,
   GoogleDriveSyncToken,
   GoogleDriveWebhook,
-  ModelId,
-  sequelize_conn,
-} from "@connectors/lib/models.js";
```

Comment:
Do you know why some imports have a .js while the file was a ts? 🙏🏻

Reply:
That is pretty weird because IIRC we never wrote JS in `connectors`